doc/source/getting-started/basic.rst

```diff
@@ -1,6 +1,5 @@
-############################
 PyAnsys project organization
-############################
+============================
 
 The `PyAnsys Project <https://docs.pyansys.com/>`_ is hosted on GitHub at `PyAnsys
```

Comment:
```suggestion
The `PyAnsys project <https://docs.pyansys.com/>`_ is hosted on GitHub at `PyAnsys
```
If you want PyAnsys Project as a proper name, then the section heading should be PyAnsys Project organization. You would then likely need to add "Project" to Vale's accept list.

Reply:
Let's got with the `project` word. The proper name is `PyAnsys`, not `PyAnsys Project`.

Reply:
I fixed this name in other places of the document.